src/config/config.yml

```diff
@@ -16,6 +16,15 @@ functions:
     method: PUT
     path: /vehicles/update-status/:systemNumber
     function: updateTechRecordStatus
+- updateEuVehicleCategory:
+    method: PUT
+    path: /vehicles/update-eu-vehicle-category/:systemNumber
+    function: updateEuVehicleCategory
+s3:
+  local:
+    endpoint: http://localhost:7000
+    s3ForcePathStyle: true
+  remote: {}
```

Comment:
I think this S3 config should be removed also

Reply:
fixed